internal/signerverifier/git/git_test.go

```diff
@@ -19,7 +19,10 @@ func TestLoadSignerFromGitConfig(t *testing.T) {
 		tmpDir := t.TempDir()
 		repo := gitinterface.CreateTestGitRepository(t, tmpDir, false)
 
-		if err := repo.SetGitConfig("gpg.format", ""); err != nil {
+		// If the local machine has a format set, setting it with an
+		// empty string would not overwrite the existing configuration.
+		// Setting it to "gpg" is equivalent to leaving it unspecified.
+		if err := repo.SetGitConfig("gpg.format", "gpg"); err != nil {
 			t.Fatal(err)
 		}
 		if err := repo.SetGitConfig("user.signingkey", ""); err != nil {
```

Comment:
do we have a similar issue with this one?

Reply:
Resolved in https://github.com/gittuf/gittuf/pull/1122/commits/0efb184bec42dd4ac96ba43b6cab25b71a55dcf7

I have attached the explanation of the issue in the commit comment.